bundle/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,375 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+     [
+       {
+         "apiVersion": "operator.nsx.vmware.com/v1",
+         "kind": "NcpInstall",
+         "metadata": {
+           "name": "ncp-install"
+         },
+         "spec": {
+           "ncpReplicas": 1
+         }
+       } 
+     ]
+    capabilities: Seamless Upgrades
+    categories: Networking, Security
+    certified: "True"
+    containerImage: vmware/nsx-container-plugin-operator:0.1.0
```

Comment:
This is not specific to this PR, as probably it was copied over from ./olm-catalog, but it might need an update

Reply:
> This is not specific to this PR, as probably it was copied over from ./olm-catalog, but it might need an update

Already update this to `vmware/nsx-container-plugin-operator:3.2.0`